app/controllers/concerns/personal_key_concern.rb

```diff
@@ -1,12 +1,12 @@
-module RecoveryCodeConcern
+module PersonalKeyConcern
   delegate :active_profile, to: :current_user
 
   def create_new_code
     if active_profile.present?
       Pii::ReEncryptor.new(user: current_user, user_session: user_session).perform
-      active_profile.recovery_code
+      active_profile.personal_key
     else
```

Comment:
I commented on the wrong line. It's the `active_profile.personal_key` that isn't tested.